BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -2,21 +2,21 @@ FullPageFormView = OrchestraView.extend(
   el: '#content'
 
   initialize: (options) ->
+    @initializer options
+    @loadTemplates [
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/fullPagePanelView',
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/elementTitle',
+    ]
+    return
+
+  initializer: (options) ->
     @options = options
     @options.listUrl = appRouter.generateUrl('listEntities', entityType: options.entityType) if options.listUrl == undefined
     @completeOptions(@options.element) if @options.element != undefined
     @events = @events || {}
-    @loadTemplates [
-      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/fullPageFormView',
```

Comment:
I don't understand : you remove the loading of the fullPageFormView template from the fullPageForView backbone view ? Is it what you want or a mistake ?